pkg/dataconn/wire.go

```diff
@@ -23,22 +24,27 @@ func NewWire(conn net.Conn) *Wire {
 }
```

Comment:
Put the header size calculation & the header buffer init here

Reply:
Updated. Thank you. The performance (latency) is improved more.